package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@linn-it/linn-form-components-library",
-    "version": "10.3.3",
+    "version": "10.3.31",
```

Comment:
should this not be 10.3.4

Reply:
yeah probably, I was just adding numbers to test it with npm pack then it was such a tiny change I left it but I'll make it .4

Reply:
@richardiphillips Fixed that - could you approve this if you get a chance?